modules/base/src/algorithm/mesh/meshclipping.cpp

```diff
@@ -486,10 +492,8 @@ std::shared_ptr<Mesh> clipMeshAgainstPlane(const Mesh& mesh, const Plane& worldS
                                                const std::vector<uint32_t>& indices,
                                                const std::vector<float>& weights) {
                         return static_cast<ValueType>(std::inner_product(
-                            indices.begin(), indices.end(), weights.begin(), T{0}, std::plus<>{},
-                            [&](uint32_t index, float weight) {
-                                return static_cast<T>(buffer[index]) * weight;
-                            }));
+                            indices.begin(), indices.end(), weights.begin(), T{0}, std::plus<T>{},
```

Comment:
did you try removing the `static`from `static const auto mix`?
Should also use [[maybe_unused]] instead of the void cast.

Reply:
yes, did not fix the problem. The only way around was to not use `buffer` in the lambda. Capturing `&buffer` led to the same problem.